navapbc/wic-mt-demo-project-eligibility-screener
# Move mockData out of test helpers and into /app/lib

## Ticket

- N/A

## Changes
> What was added, updated, or removed in this PR.

- Move /app/tests/helpers/mockData.ts to /app/lib/mockData.ts

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

This PR moves /app/tests/helpers/mockData.ts to /app/lib/mockData.ts. This file was originally created as a test helper, but then it became also useful as a helper for storybook (see [discussion](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/142#discussion_r1009611575)). It makes more sense to be in a shared app dir, rather than in the tests dir.

## Testing
> Screenshots, GIF demos, code examples or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

First test: Run `yarn test` to make sure there are no regressions.

Second test: Check the storybook still works as expected:
  1. Checkout this branch
  2. Build and run the container: `docker-compose up -d --build`
  4. Build and run storybook: `docker-compose exec nextjs yarn storybook`
  5. Navigate to localhost:6006
  7. Remove the container when you're done: `docker-compose down`
